be/src/exec/sorting/merge_column.cpp

```diff
@@ -386,6 +386,16 @@ int SortedRun::compare_row(const SortDescs& desc, const SortedRun& rhs, size_t l
     return 0;
 }
 
+bool SortedRun::is_sorted(const SortDescs& desc) const {
```

Comment:
http://0x80.pl/notesen/2018-04-11-simd-is-sorted.html

Reply:
It's only for DCHECK now.